src/fights.ts

```diff
@@ -225,7 +225,7 @@ function embezzlerSetup() {
     });
   }
 
-  if (!get("_enamorangs") && !itemAmount($item`LOV Enamorang`) && averageEmbezzlerNet > 20000) {
+  if (!get("_enamorangs") && !have($item`LOV Enamorang`) && averageEmbezzlerNet > 20000) {
```

Comment:
we actually want to use itemAmount here; itemAmount only checks the inventory, while have will sometimes check the closet